README.md

```diff
@@ -277,13 +277,13 @@ not working correctly becuase GatorGrader could not be pulled correctly.
 The main reason for this issue is out of date dependency versions not
 lining up with the current release of GatorGrader. However, there
 have beeen some instances of this issue without a clear explanation.
-To address this issue locaclly, locate the `virtualenvs` and `gatorgrader`
+To address this issue locally, locate the `virtualenvs` and `gatorgrader`
 directories through the `.local/share` path. Delete `virtualenvs` and
 `gatorgrader` folders and their contents. To address this issue in a container,
 clear the DockaGator cache using the `docker volume rm dockagator` command.
-After deleting these folders,
-re-build your virtual envirnment to re-generate GatorGrader on your device by
-either running `gradle grade` or a Dockagator command.
+After deleting these folders, rebuild your virtual environment on your device by
+either running `gradle grade` or a
+[DockaGator](https://github.com/GatorEducator/dockagator)command.
```

Comment:
Sorry for all the nitpicking... but you're missing a space before `command`. Also, it may read slightly better if you use `the virtual environment` instead of `your virtual environment`.